### PR TITLE
Improving styling to match HomeAssistant entities card

### PR DESCRIPTION
**Pull request by afarago**
This PR is a styling upgrade to match the styling of HomeAssistant entities card.

* icons using the standard HA iconset via on demand and thus with a graceful degradation (no icon shown if WAN is down)
* group by entity_categories
* instead of table use div and grid/flex layouts
* added uptime based on ping value
* color picker for rgb lights
* Tag column hidden on web view
* update icon to use the 

Icons and entity_categories functionally needs matching PR in esphome/web_server component - though the two still work in the previous way without the other.

Not sure about coding standards, or formatting in this repo. Please comment and help me with a pointer if styling changes are needed. 

Related discord discussion - https://discord.com/channels/429907082951524364/1177062706340171807

### CODE REVIEW

**Comment by afarago** [comment]
Pull request in [esphome](https://github.com/esphome/esphome): esphome/esphome#5854

**Review by jesserockz — CHANGES_REQUESTED** [review]
I just have one comment to make as I am not very familiar with typescript/lit and frontend in general

File: .vscode/tasks.json
[the remark this answers is not in the document]
Comment (afarago) [reply .vscode/tasks.json:1]: OK, thanks for catching this mistake.

**Comment by afarago** [comment]
I believe I have marked the requested change resolved, yet am a bit unsure over the github status.
Please let me know if there are any pending actions on my side.

**Comment by DocBrown101** [comment]
Can I help with the review, or what exactly is still missing here?

**Comment by kevireilly** [comment]
@DocBrown101 it has been discussed in the discord [thread](https://discord.com/channels/429907082951524364/1177062706340171807) that this work would likely be best represented as v3 instead of modifying v2. However, in order to facilitate that, there are some structural changes required via #53.

Once that lands, this work can be rebased and restructured using a new v3 package and then we should be clear for lift off on merging, deploying, and beta testing with those tuned in.

**Comment by kevireilly** [comment]
This pull request can probably be closed as #54 supersedes it.

**Comment by DocBrown101** [comment]
From which version will it be available as v3 in an esphome release?

**Pull request closed without merging**